packages/react-components/src/localization/locales/en-US/strings.json

```
@@ -254,6 +254,11 @@
     "secondaryText": "Please update your browser and then try joining the call again.",
     "moreHelpText": "See compatibility requirements"
   },
+  "UnsupportedOperatingSystem": {
+    "primaryText": "Operating system not supported",
+    "secondaryText": "Please join this call using a device with a compatible operating system.",
+    "moreHelpText": "See compatibility requirements"
```

highlight this is a button/link:
```suggestion
    "moreHelpLinkText": "See compatibility requirements"
```
Unless we specifically do not do that throughtout the strings file, consistency is better